tox.ini

```diff
@@ -1,18 +1,35 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37,pypy,pypy3,coverage
+    py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
+# Note: if you add new Python versions, please add them to
+# [testenv:coverage] depends as well!
 
 [testenv]
 commands =
-    zope-testrunner --test-path=src []
-extras = test
+    coverage run -p -m zope.testrunner --test-path=src []
+extras =
+    test
+deps =
+    coverage
+setenv =
+     ZOPE_INTERFACE_STRICT_IRO=1
```

Comment:
The indentation, it is not consistent, and I can't help noticing....

Reply:
You caught me again! Thank you.